python_modules/dagster/dagster/_core/definitions/declarative_automation/automation_context.py

```diff
@@ -80,6 +82,7 @@ def create(key: EntityKey, evaluator: "AutomationConditionEvaluator") -> "Automa
             request_subsets_by_key=evaluator.request_subsets_by_key,
             parent_context=None,
             _cursor=evaluator.cursor.get_previous_condition_cursor(key),
+            _daemon_cursor=evaluator.cursor,
```

Comment:
All_assets_cursor? Full_cursor? “Daemon” doesn’t convey that it has all of them to me